### DISCUSSION TITLE
Osu layer on Samsung Dex (Android) doesn´t recognise mouse input.

### DISCUSSION THREAD

**Discussion by LeonLikesCats** [ppy/osu#25138] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Hello, 
i want to report that Osu Lazer on Android does not recognise the mouse input while using Samsung Dex. The S Pen works but not the mouse input. In tablet mode everything works how it should be. 

### Screenshots or videos

https://github.com/ppy/osu/assets/147930977/2686cb81-cc87-47af-9eea-51ece9b1d7d8

### Version

Osu 2023.1008.1-lazer

### Logs

I don´t have acces to it

**Comment by NamOpt** [comment]
pls DON'T using Samsung Dex for play osu!lazer, it's not support in osu!lazer, because Samsung Dex was born to connect your phone to a large screen to work and it's not good for playing games, moreover osu!lazer on android only recognizes touch, not mouse.

**Reply by EDLLT** [reply]
You are mistaken, OSU on android DOES recognize the mouse and the keyboard (you can use the mouse and keyboard with no limitations whatsoever) but ONLY if you aren't in Dex mode.
Going into Dex mode breaks that functionality. I believe it's a really simple bug fix

**Comment by Joehuu** [comment, marked as answer]
Seems like https://github.com/ppy/osu-framework/pull/6105 will fix this. Have tested on S10e.
